components/consentRequestForm/index.jsx

```diff
@@ -325,6 +345,7 @@ export default function ConsentRequestForm({ agentDetails, agentWebId }) {
 }
 
 ConsentRequestForm.propTypes = {
+  consentRequest: T.objectOf(T.any).isRequired,
```

Comment:
It'd probably be ideal to use a proper `T.shape` here instead of just `T.object(T.any)`, though you could definitely make that shape just include the properties you use, I think.